src/ontology/mondo-ingest.Makefile

```diff
@@ -18,7 +18,7 @@ SKIP_HUGE=false
 ####################################
 .PHONY: pip-%
 pip-%:
-	 python3 -m pip install --upgrade $*
+	 python3 -m pip install --upgrade $* --break-system-packages
```

Comment:
## `--break-system-packages`
> The `--break-system-packages` flag in pip is a potentially dangerous option that allows pip to modify or replace packages that are part of your system's Python installation.

What is the rationale for using this here? Is this a temporary change?

If it is a permanent change, does it need to be bundled in this PR or can it be added separately?

Reply:
In the future this should not be necessary anymore, as we should rarely if ever need to do pip install. Unfortunately, this is a sideeffect of migrating to python 3.12.. its just needed now to add this option if you are installing into the global (rather than a virtual) environment. the next odk version (already on `dev`) will be on python 3.12 and need this.